docs/configuration.rst

```diff
@@ -63,3 +68,67 @@ Note that, equivalently, you can add these during webapp configuration:
             # ...and so on with the other settings...
             config = Configurator(settings=settings)
             config.include('pyramid_swagger')
+
+
+generate_resource_listing
+-------------------------
+
+With a large API (many Resource objects) the boilerplate ``apis`` field of
+the `Resource Listing`_ document can become painful to maintain. This
+setting provides a way to relieve that burden.
+
+When the ``generate_resource_listing`` option is enabled
+:mod:`pyramid_swagger` will automatically generate the ``apis`` section of
+the swagger `Resource Listing`_ from the list of ``*.json`` files in the
+schema directory. The ``apis`` listing is generated by using the name of the
+file (without the extension) as the ``path``.
+
+To use this feature, create an ``api_docs.json`` file in the schema directory.
+This file may contain any relevant field from `Resource Listing`_,
+but it **must** exclude the ``apis`` field. In many cases this
+``api_docs.json`` will only contain a single key ``swaggerVersion: 1.2``.
+
+.. _Resource Listing: https://github.com/swagger-api/swagger-spec/blob/master/versions/1.2.md#user-content-51-resource-listing
+
+Example
+~~~~~~~
+
+Given a schema directory with the following files
+
+.. code-block:: none
+
+    api_docs/
+    ├── api_docs.json
+    ├── pet.json
+    ├── store.json
+    └── user.json
+
+Previously you might have created an ``api_docs.json`` that looked like this
+
+.. code-block:: json
+
+    {
+        "swaggerVersion": "1.2",
+        "apiVersion": "1.0",
+        "apis": [
+            {
+                "path": "/pet",
+            },
+            {
+                "path": "/store",
+            },
+            {
+                "path": "/user",
+            },
+        ]
+    }
+
+When ``generate_resource_listing`` is enabled, the ``api_docs.json`` should
+be similar, but with the ``apis`` section removed.
+
+.. code-block:: json
+
+    {
+        "swaggerVersion": "1.2",
+        "apiVersion": "1.0",
+    }
```

Comment:
And just to be explicit, do you want to specify what this will generate? Or leave it a little vague so you aren't bound to a particular implementation? Worth calling out that this will not generate any description strings? https://github.com/swagger-api/swagger-spec/blob/master/versions/1.2.md#512-resource-object